MMA.md

```diff
@@ -0,0 +1,365 @@
+~~~~
+{
+  "time": "1500062287.263575",
+  "client": "ios",
+  "token": "yFmAZ4G4TReVcJY8rjhtVudo98xHyEELsIlbSM2UcaY",
+  "event": "E_First_Run",
+  "userId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "value": "0.000000",
+  "action": "track",
+  "devMode": true,
+  "deviceId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "sdkVersion": "2.0.0-SNAPSHOT"
+}
+~~~~
+
+The second launch after install
+~~~~
+{
+  "time": "1500062287.263575",
+  "client": "ios",
+  "token": "yFmAZ4G4TReVcJY8rjhtVudo98xHyEELsIlbSM2UcaY",
+  "event": "E_Second_Run",
+  "userId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "value": "0.000000",
+  "action": "track",
+  "devMode": true,
+  "deviceId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "sdkVersion": "2.0.0-SNAPSHOT"
+}
+~~~~
+
+Whenever the App is launched.
+~~~~
+{
+  "time": "1500062287.263575",
+  "client": "ios",
+  "token": "yFmAZ4G4TReVcJY8rjhtVudo98xHyEELsIlbSM2UcaY",
+  "event": "E_Opened_App",
+  "userId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "value": "0.000000",
+  "action": "track",
+  "devMode": true,
+  "deviceId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "sdkVersion": "2.0.0-SNAPSHOT"
+}
+~~~~
+
+The user loads a bookmark from home panel.
+~~~~
+{
+  "time": "1500062287.263575",
+  "client": "ios",
+  "token": "yFmAZ4G4TReVcJY8rjhtVudo98xHyEELsIlbSM2UcaY",
+  "event": "E_Opened_Bookmark",
+  "userId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "value": "0.000000",
+  "action": "track",
+  "devMode": true,
+  "deviceId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "sdkVersion": "2.0.0-SNAPSHOT"
+}
+~~~~
+
+The user opened a new tab.
+~~~~
+{
+  "time": "1500062287.263575",
+  "client": "ios",
+  "token": "yFmAZ4G4TReVcJY8rjhtVudo98xHyEELsIlbSM2UcaY",
+  "event": "E_Opened_New_Tab",
+  "userId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "value": "0.000000",
+  "action": "track",
+  "devMode": true,
+  "deviceId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "sdkVersion": "2.0.0-SNAPSHOT"
+}
+~~~~
+
+The user interacts with search url area.
+~~~~
+{
+  "time": "1500062287.263575",
+  "client": "ios",
+  "token": "yFmAZ4G4TReVcJY8rjhtVudo98xHyEELsIlbSM2UcaY",
+  "event": "E_Interact_With_Search_URL_Area",
+  "userId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "value": "0.000000",
+  "action": "track",
+  "devMode": true,
+  "deviceId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "sdkVersion": "2.0.0-SNAPSHOT"
+}
+~~~~
+
+The user saves a bookmark.
+~~~~
+{
+  "time": "1500062287.263575",
+  "client": "ios",
+  "token": "yFmAZ4G4TReVcJY8rjhtVudo98xHyEELsIlbSM2UcaY",
+  "event": "E_Saved_Bookmark",
+  "userId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "value": "0.000000",
+  "action": "track",
+  "devMode": true,
+  "deviceId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "sdkVersion": "2.0.0-SNAPSHOT"
+}
+~~~~
+
+The user opened a telephone link from a web page.
+~~~~
+{
+  "time": "1500062287.263575",
+  "client": "ios",
+  "token": "yFmAZ4G4TReVcJY8rjhtVudo98xHyEELsIlbSM2UcaY",
```

Comment:
Is this our API token? Not sure how to remove this from history but please remove that from this commit!